pipeline_tools/confirm_submission.py

```diff
@@ -59,7 +59,10 @@ def confirm(envelope_url, http_requests):
     headers = {
         'Content-type': 'application/json'
     }
-    response = http_requests.put('{}/submissionEvent'.format(envelope_url), headers=headers)
+    response = http_requests.put('{}/submissionEvent'.format(envelope_url),
+                                 headers=headers,
+                                 retry=retry_if_exception_type(requests.exceptions.ReadTimeout)
```

Comment:
In these requests, we still want to retry if the exception type is `requests.HTTPError` or `requests.ConnectionError` as well. Right now this overrides that default: https://github.com/HumanCellAtlas/pipeline-tools/blob/master/pipeline_tools/http_requests.py#L146

How about including `requests.exceptions.ReadTimeout` as a retryable status code here? https://github.com/HumanCellAtlas/pipeline-tools/blob/master/pipeline_tools/http_requests.py#L143

Reply:
As discussed in person, this actually does not override the default retry parameter! But it would still be simpler to include `requests.exceptions.ReadTimeout` as a default retryable error so that it does not have to be passed into each request.